src/validation/Validator.ts

```diff
@@ -495,16 +495,16 @@ export class Validator {
      * Checks if the string contains only letters (a-zA-Z).
      * If given value is not a string, then it returns false.
      */
-    isAlpha(value: string): boolean {
-        return typeof value === "string" && this.validatorJs.isAlpha(value);
+    isAlpha(value: string, locale: ValidatorJS.AlphaLocale = "en-US"): boolean {
```

Comment:
Please leave locale parameter without default value. `validator.js` lib will use "en-US" by default. Same for isAlphanumeric.